src/assets/locales/en.json

```diff
@@ -72,6 +72,7 @@
   "close-room.message": "Closing this room will remove yourself and others from the room, shutting it down permanently.\n\nAre you sure? This action cannot be undone.",
   "close-room.confirm": "Yes, Close Room",
   "close-room.cancel": "Cancel",
+  "close-room.type-to-confirm": "Type room name to confirm:",
```

Comment:
You'll need to run the `sync-locales.js` script to update the other locales with this new string.